src/component/registry/bpmn-elements-registry.ts

```diff
@@ -61,7 +62,10 @@ export class BpmnElementsRegistry {
     private cssRegistry: CssRegistry,
     private graphCellUpdater: GraphCellUpdater,
   ) {
-    this.bpmnModelRegistry.registerOnLoadCallback(this.cssRegistry.clear);
+    this.bpmnModelRegistry.registerOnLoadCallback(() => {
+      this.cssRegistry.clear();
+      this.graphCellUpdater.styleManager.clear();
```

Comment:
**nitpick:** there is no tests that show that the `StyleManager` needs to be cleared.
If I comment the line, all tests (unit and integration ) pass.
If I comment the `CssRegistry` clear, some integration tests fail.

Reply:
I'll add the missing test with 2 different diagram, with some common id on different elements.